src/bundle/Resources/public/js/scripts/admin.distraction.free.mode.js

```diff
@@ -16,6 +17,43 @@
         activeFieldEdit.classList.toggle('ibexa-field-edit--distraction-free-mode-active', active);
         editorInstance.set('distractionFreeModeActive', active);
 
+        if (active) {
+            let parentElement = activeFieldEdit.parentNode;
+
+            while (parentElement && parentElement !== doc.body) {
```

Comment:
I think this new fragment may be more readable if split into two functions e.g. `resetAncestorsPosition()`, `restoreAncestorsPosition()`